### ISSUE TITLE
xActiveDirectory: Move the examples in the README.md to Examples folder

### ISSUE THREAD

**Issue by johlju** [PowerShell/xActiveDirectory#263]
There are examples in the README.md. These examples should be moved out and added as example files under individual resource folder in the https://github.com/PowerShell/xActiveDirectory/tree/dev/Examples folder. 

Moving the examples out to Examples folder allows the common test to validate the examples.

**Comment by SSvilen** [comment]
What about the examples , which include multiple resources?

**Comment by johlju** [comment]
Either we split them up into individual examples. But if they are meant to describe a bigger scenario, then maybe we put them directly under the Examples folder and not in a individual resource example folder. 🤔 

I'm okay with examples is using other resources in the example as well if that helps describe the examples, even if that means examples are duplicated. 🤔 

If there are examples not part of a particular resource folder, then those examples could be listed in the examples [README.md](https://github.com/PowerShell/xActiveDirectory/tree/dev/Examples), and the example files can be put somewhere suitable. 🙂 

Basically, do what you feel is best. 😃

**Comment by SSvilen** [comment]
Cool thanks. I'll submit a PR later this week and then will work those cases out.
